docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+
+#### For all tests:
+- Change ID 9b829667f797879a0a4cd911eb08c4808e0f8083 (Thu Feb 4 13:38:37 2021)
+- 8 Node (1.18, nodes: 4vcpu 16Gb)
+- Envoy proxy log level Error
+- OSM controller log level Trace (reasons)
+- HTTP debug server disabled on OSM
+- Test topology deploys each iteration:
+	- 2 clients
+		- 5 replicaset per client
+	- 5 server services
+		- 2 replicaset per server service
+	- 1 TrafficSplitt between the 5 server services (10 pods backed)
+	- Total of 20 pods each iteration. 10 client pods will REST GET the Traffic split.
+	- Correctness is ensured. It is checked that all TrafficSplitt server members are eventually reached.
+- Test timeout for network correctness: 150 seconds
+
+#### Varying form factors
+- with 256MB max RAM
+	- 1.5 CPU
+- With 512MB RAM
+	- 1.5 CPU
+	- 4 CPU
+
+(All tests were run multiple times to ensure validity of the results)
+
+## Assessment
+
+Note: Please assume we are deploying a proxy per pod in the following results, so use pod/proxies interchangeably.
+### Memory
+- Memory per pod/envoy onboarded in the network is calculated after the initial snapshot with nothing onboarded on the mesh is seen to take into account standalone memory used by OSM.
+  - Initial memory use of OSM **40~46MB**
+  - Memory increase per pod ranges between **800KB~1MB** per pod. 
+- With **256MB** max memory limit for OSM's container, **180-200 pods** should be a conservative limit. Any more pods
+can (and will) have OSM `OOMKilled` by Kubernetes.
+- With **512MB** max memory limit for OSM's container, it can reach to **~400 pods**, although convergence times will spike through the roof up to **+120s** with a rather oversubscribed CPU profile, that could potentially lead to other issues.
+
+### CPU
+#### 1.5/2 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **10s-20s** and full configuration convergence ranges around **20-40s**
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **40-90s** and full configuration convergence ranges around **+150s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+#### 4 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **5-10s**, however full configuration convergence falls into the **20-40s** bucket.
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **20-40s** and full configuration convergence ranges around **+90s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+
+- In both cases, and taking into account the architecture limitations explained above, OSM will run hot on CPU, most of the time recomputing ADS updates for all pods in the mesh. This oversubscription could lead to not enough cycles available for other critical path routines (liveness), depending entirely on the scheduler.
```

Comment:
Very interested in seeing the difference in performance if we do not recompute ADS updates for all pods.

Reply:
remind me to give to show you the POC offline :)